lib/src/main/kotlin/com/outdatedversion/survival/command/TimeStampCommand.kt

```diff
@@ -0,0 +1,55 @@
+package com.outdatedversion.survival.command
+
+import co.aikar.commands.BaseCommand
+import co.aikar.commands.annotation.CommandAlias
+import co.aikar.commands.annotation.CommandCompletion
+import co.aikar.commands.annotation.Default
+import com.outdatedversion.survival.Plugin
+import net.md_5.bungee.api.ChatColor
+import org.bukkit.Bukkit
+import org.bukkit.NamespacedKey
+import org.bukkit.entity.Player
+import org.bukkit.persistence.PersistentDataType
+import java.time.ZoneId
+import java.util.*
+// Now using Annotation Command Framework (ACF)
+@CommandAlias("timezone|tz")
+class TimeStampCommand(main: Plugin): BaseCommand() {
```

Comment:
there is a shortcut for member initialization
```suggestion
class TimeStampCommand(private val plugin: Plugin): BaseCommand() {
```

Reply:
Done! Will take effect next commit!